README.md

```diff
@@ -40,10 +40,10 @@ Vagrant
 -------
 
 The setup that most developers are using is `vagrant-libvirt` with
-`vagrant-hostmanager` installed on Fedora via:
+`vagrant-hostmanager` and `vagrant-sshfs` installed on Fedora via:
```

Comment:
Vagrant can be installed on CentOS/RHEL 7,8 from https://www.vagrantup.com/downloads.html too

Please add here note that plugins can be installed using:

```
vagrant plugin install vagrant-hostmanager
vagrant plugin install vagrant-libvirt
vagrant plugin install vagrant-sshfs
```

, when CentOS/RHEL is used.